CMakeLists.txt.in

```diff
@@ -83,7 +83,7 @@ ExternalProject_Add(protobuf
 set(dpdk_quadruple ${CMAKE_SYSTEM_PROCESSOR}-native-linuxapp-gcc)
 
 set (dpdk_args
-  EXTRA_CFLAGS=-Wno-error
+  "EXTRA_CFLAGS=-Wno-error -fcommon"
```

Comment:
what does `-fcommon` do 

this seems OK, but typically a change in flags, would be it's own commit.

Reply:
You`re right, that actually slipped inside the commit from redpanda, I cleaned that up with few other bits in [https://github.com/smfrpc/smf/pull/375/commits/713a5ae4c7519198d0c537ad1f0fec06695e6642](https://github.com/smfrpc/smf/pull/375/commits/713a5ae4c7519198d0c537ad1f0fec06695e6642)

It`s just a performance flag for the linker [https://man7.org/linux/man-pages/man1/gcc.1.html](https://man7.org/linux/man-pages/man1/gcc.1.html), not really required imho.